.github/workflows/lint.yml

```
@@ -15,14 +15,17 @@ jobs:
   golangci:
     strategy:
       matrix:
-        module: ["chain", "cosmosutil", "node", "types", "util", "wallet"]
+        module: ["cosmos", "general"]
```

```suggestion
        module: ["cosmos", "core"]
```

I would rename the `general` package to `core`.  I think that is a bit clearer and standard

---

i like that